src/components/WeeklySummary/__mocks__/weeklySummaryMockData.js

```diff
@@ -0,0 +1,45 @@
+
+import moment from 'moment';
+import 'moment-timezone';
+
+const dueDateThisWeek = moment().tz('America/Los_Angeles').endOf('week').toISOString();
+const dueDateLastWeek = moment().tz('America/Los_Angeles').endOf('week').subtract(1, 'week')
+  .toISOString();
+const dueDateBeforeLast = moment().tz('America/Los_Angeles').endOf('week').subtract(2, 'week')
+  .toISOString();
```

Comment:
Here you set the time-zone to 'America/Los_Angeles'. Should this be set as a standard in global?

Reply:
Thanks Adam, originally it was supposed to be local time to the user but after discussion with Jae he wants it LA time. If we did local, things can get really complicated, we ran a few scenarios at the time of discussion but keeping it LA time would be the simpler approach which works for Jae as well. Keep in mind that regardless of which timezone you start the week, it will end in that same timezone so you're not going to lose or gain any time, it's the same. Just the countdown timer will be showing you the time left relative to LA time.